### ISSUE TITLE
Capability to distinguish .Net Framework 4.8 and .Net Framework 4.8.1

### ISSUE THREAD

**Issue by aries-zhang** [EliteLoser/DotNetVersionLister#17]
At the moment, the `Get-STDotNetVersion` command seems to print the same output as following no matter whether .Net Framework 4.8 or .Net Framework 4.8.1 is installed.

```
ComputerName : localhost
>=4.x        : 4.8 or later
v4\Client    : Installed
v4\Full      : Installed
v3.5         : Not installed (no key)
v3.0         : Not installed (no key)
v2.0.50727   : Not installed (no key)
v1.1.4322    : Not installed (no key)
Ping         : True
Error        :

    Script start time: 11/16/2022 20:31:15
    Script end time:   11/16/2022 20:31:15
```

Request to have the deterministic result for .Net Framework 4.8 and .Net Framework 4.8.1.

**Comment by EliteLoser** [comment]
Hi. The version here on GitHub differentiates 4.8 and 4.8.1.

I got a pull request with some refactored code that seemed like an improvement, so I merged it and adapted it a bit.

The code will now display `4.8.1` for that version and `4.8.1+` if there's a higher version. The "or later" has turned into a more precise "later" with the "+" at the end.

Anyway, if you git clone or download from GitHub somehow, the latest version here should work for your case.

I don't have access to a Windows computer right now, or I would push the new version to the PSGallery. That is now a minor version behind the version here on Github.
